build/src/wrapper.c

```diff
@@ -424,3 +424,7 @@ int EXPORT(deserialize) (const char* schema, void* data, int bytes, int flags) {
   );
   return last_status;
 }
+
+int EXPORT(get_autocommit) () {
```

Comment:
I think calling this just `autocommit` is fine 😅

Also, can we move this next to the functions for changes and total changes + add a short comment about what it does.